src/main/java/com/google/api/codegen/viewmodel/OptionalArrayMethodView.java

```diff
@@ -57,6 +57,8 @@
 
   public abstract boolean isLongrunningOperation();
 
+  public abstract boolean hasLongrunningReturnValue();
```

Comment:
My concern is having multiple places to specify the same thing. It could get confusing for another language implementing LRO features, even if we currently have it clear in the existing languages.